app/views/examples/translated/index.njk

```diff
@@ -0,0 +1,871 @@
+          type: "button"
+        }
+        ]
+      }
+    ]
+  }) }}
+
+  {{ govukDateInput({
+    id: "passport-issued",
+    namePrefix: "passport-issued",
+    fieldset: {
+      legend: {
+        text: "Pryd gafodd eich pasbort ei gyhoeddi?",
+        isPageHeading: true,
+        classes: "govuk-fieldset__legend--l"
+      }
+    },
+    hint: {
+      text: "Er enghraifft, 27 3 2007"
+    },
+    items: [
+      {
+        name: "day",
+        label: "Dydd",
+        classes: "govuk-input--width-2",
+        autocomplete: "bday-day"
+      },
+      {
+        name: "month",
+        label: "Mis",
+        classes: "govuk-input--width-2",
+        autocomplete: "bday-month"
+      },
+      {
+        name: "year",
+        label: "Blwyddyn",
+        classes: "govuk-input--width-4",
+        autocomplete: "bday-year"
+      }
+    ]
+  }) }}
+
+  {{ govukDetails({
+    summaryText: "Help gyda chenedligrwydd",
+    text: "Mae angen i ni wybod eich cenedligrwydd fel y gallwn weithio allan pa etholiadau y mae gennych hawl i bleidleisio ynddynt. Os na allwch nodi eich cenedligrwydd, bydd yn rhaid i chi anfon copïau o ddogfennau adnabod drwy'r post."
+  }) }}
+
+  {{ govukErrorSummary({
+    titleText: "Mae yna broblem",
+    disableAutoFocus: true,
+    errorList: [
+      {
+        text: "Rhaid i'r dyddiad y cyhoeddwyd eich pasbort fod yn y gorffennol",
+        href: "#passport-issued-day"
+      },
+      {
+        text: "Rhowch god post, fel AA1 1AA",
+        href: "#postcode-input"
+      }
+    ]
+  }) }}
+
+  {{ govukFileUpload({
+    id: "file-upload-1",
+    name: "file-upload-1",
+    label: {
+      text: "Llwythwch ffeil i fyny"
+    }
+  }) }}
+
+  {{ govukFooter({
+    classes: "govuk-!-margin-bottom-4",
+    navigation: [
+      {
+        title: "Gwasanaethau a gwybodaeth",
+        width: "two-thirds",
+        columns: 2,
+        items: [
+          {
+            href: "#",
+            text: "Budd-daliadau"
+          },
+          {
+            href: "#",
+            text: "Genedigaethau, marwolaethau, priodasau a gofal"
+          },
+          {
+            href: "#",
+            text: "Busnes a hunan-gyflogedig"
+          }
+        ]
+      },
+      {
+        title: "Adrannau a pholisi",
+        width: "one-third",
+        items: [
+          {
+            href: "#",
+            text: "Sut mae llywodraeth yn gweithio"
+          },
+          {
+            href: "#",
+            text: "Adrannau"
+          },
+          {
+            href: "#",
+            text: "Ar draws y byd"
+          }
+        ]
+      }
+    ],
+    meta: {
+      items: [
+        {
+          href: "#",
+          text: "Cwcis"
+        },
+        {
+          href: "#",
+          text: "Cysylltwch"
+        },
+        {
+          href: "#",
+          text: "Telerau ac Amodau"
+        }
+      ],
+      html: 'Adeiladwyd gan y <a href="#" class="govuk-footer__link">Government Digital Service</a>'
+    },
```

Comment:
I think you also want to set `meta.visuallyHiddenTitle` here to change the hidden H2 from it's default 'Support Links'

https://github.com/alphagov/govuk-frontend/blob/aafbb16079a53730ba9a8a0e2cc0aed647237fe8/src/govuk/components/footer/template.njk#L35